components/footer.tsx

```diff
@@ -65,10 +65,10 @@ export default function Footer() {
 
           <div className="sm:col-span-6 md:col-span-4 lg:col-span-3">
             <h6 className="text-primary-300 font-medium mb-2">Find us on CNCF Landscape</h6>
-            <Link href="https://landscape.cncf.io/card-mode?selected=keploy">
+            <Link href=" https://landscape.cncf.io/?item=app-definition-and-development--continuous-integration-delivery--keploy" target="_blank">
```

Comment:
LGMT ! There's a extra space can you remove that

Reply:
@Sonichigo I think we are good to go, I have removed the space.